.github/workflows/build.yml

```diff
@@ -12,6 +12,8 @@ jobs:
         uses: actions/checkout@v2
       - name: update
         run: |
+          # note the output of git submodule status varies pre/post init
+          git submodule init --recursive
```

Comment:
I think you should be able to checkout with submodules:
```yaml
- uses: actions/checkout@v2
  with:
    submodules: true
```
Not that it's terribly important